### PR TITLE
Add documentation for dune utop usage in emacs

### PR DESCRIPTION
**Pull request by samarthkishor**
Addresses #306

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,13 @@ your `~/.emacs`:
 (setq utop-command "opam config exec -- utop -emacs")
 ```
 
+If you use `dune` and want to launch `dune utop` in emacs, you
+should add this to your `~/.emacs`:
+
+```scheme
```
Comment (pmetzger) [inline README.md]: I believe "Emacs Lisp" is a valid language. We should check if that's correct and probably switch to it.

Comment (samarthkishor) [reply README.md]: I went ahead and changed the Scheme code blocks to Emacs Lisp (`elisp`). The highlighting seems to look correct on Github.

**Comment by pmetzger** [comment]
@diml You okay with the changes?

**Comment by diml** [comment]
Yes, they look good to me!

**Pull request merged**